vscode/src/utilities/configuration.ts

```diff
@@ -13,18 +14,21 @@ export function getConfigKaiRpcServerPath(): string {
   return getConfigValue<string>("kaiRpcServerPath") || "";
 }
 
-export function getConfigLogLevel(): string {
-  return getConfigValue<string>("logLevel") || "debug";
+export function getConfigLogLevel(): ServerLogLevels {
+  return getConfigValue<ServerLogLevels>("logLevel") || "DEBUG";
 }
 
+// TODO: Is this used?
```

Comment:
I think this can be in the follow-up but we should absolutely be passing this onto `kai` either via their cli arguments or in the initialize call.

I do not think users should typically be messing with it but we should keep them as configurable.

Reply:
There are no cli args or initialize args that support those settings else I would have passed them along.

Reply:
I'll update descriptions and comments so they make more sense.

Reply:
Created https://github.com/konveyor/kai/issues/550 and I'll reference those in the code.